src/storybook/stories/Section.stories.tsx

```diff
@@ -0,0 +1,194 @@
+};
+export default SectionMeta;
+
+export const ListedSectionItems: Meta<SectionMetaProps> = {
+  args: {withFullPadding: true},
+  decorators: [
+    (Story, {args}) => (
+      <ScrollView>
+        <Story
+          args={{
+            ...args,
+            children: (
+              <Section>
+                <HeaderSectionItem
+                  text="HeaderSectionItem"
+                  mode="heading"
+                  transparent={false}
+                />
+                <ButtonSectionItem
+                  label="ButtonSectionItem"
+                  onPress={() => {}}
+                />
+                <ActionSectionItem
+                  text="ActionSectionItem"
+                  onPress={() => {}}
+                />
+                <CounterSectionItem
+                  text="CounterSectionItem"
+                  count={2}
+                  addCount={() => {}}
+                  removeCount={() => {}}
+                />
+                <ExpandableSectionItem
+                  text="ExpandableSectionItem"
+                  onPress={() => {}}
+                  expandContent={<ThemeText>Content</ThemeText>}
+                />
+                <FavoriteDepartureSectionItem
+                  favorite={{
+                    id: '1',
+                    stopId: '2',
+                    lineId: '3',
+                    quayId: '4',
+                    quayName: 'FavoriteDepartureSectionItem',
+                  }}
+                />
+                <FavoriteSectionItem
+                  favorite={{
+                    id: '1',
+                    location: {
+                      id: '2',
+                      name: 'FavoriteSectionItem',
+                      layer: 'address',
+                      coordinates: {longitude: 3, latitude: 4},
+                      locality: '',
+                      category: [],
+                      resultType: 'favorite',
+                      favoriteId: '5',
+                    },
+                  }}
+                />
+                <GenericClickableSectionItem>
+                  <ThemeText>GenericClickableSectionItem</ThemeText>
+                </GenericClickableSectionItem>
+                <GenericSectionItem>
+                  <ThemeText>GenericSectionItem</ThemeText>
+                </GenericSectionItem>
+                <InternalLabeledSectionItem label="InternalLabeledSectionItem">
+                  <ThemeText>Content</ThemeText>
+                </InternalLabeledSectionItem>
+                <LinkSectionItem text="LinkSectionItem" />
+                <LocationInputSectionItem
+                  label="LocationInputSectionItem"
+                  onPress={() => {}}
+                />
+                <MessageSectionItem
+                  messageType="valid"
+                  message="MessageSectionItem"
+                />
+                <PhoneInputSectionItem
+                  label="PhoneInputSectionItem"
+                  prefix="+47"
```

Comment:
Should only be `"47"` as it produces a double +:
<img width="145" alt="Skjermbilde 2024-02-29 kl  09 54 09" src="https://github.com/AtB-AS/mittatb-app/assets/134292729/fe9e40da-656b-41df-8325-39488801ae4f">